src/download.ts

```diff
@@ -432,42 +433,28 @@ export async function getFileContentHns<T = unknown>(
  * @returns - An object containing the data of the file, the content-type, metadata, and the file's skylink.
  * @throws - Will throw if the request does not succeed or the response is missing data.
  */
-export async function getFileContentRequest<T = unknown>(
+export async function getFileContentRequest(
   this: SkynetClient,
   url: string,
   customOptions?: CustomDownloadOptions
-): Promise<GetFileContentResponse<T>> {
+): Promise<AxiosResponse> {
```

Comment:
This change is breaking compatibility. Do we absolutely need to do that?
Let's, instead of breaking compat, add a new method called `getFileContentRequestAxios`?
We can even use the new method here, instead of duplicating the code, so the maintenance overhead of having both will be minimal.

Reply:
`getFileContentRequest` is a protected method so it's not exported. This change was for internal consistency since the other `*Request` methods return `Promise<AxiosResponse>` already, and also to allow us to access the headers from tests.

Reply:
https://github.com/SkynetLabs/skynet-js/blob/master/src/client.ts#L113
Sorry for the confusion, I know it's not clear from just looking at `download.ts`.

Reply:
Oh, I see. You're right, this is fine then. I saw `exported` and didn't check the chain.